spec/factories/event_schedule.rb

```diff
@@ -2,6 +2,7 @@
 
 FactoryGirl.define do
   factory :event_schedule do
+    start_time DateTime.parse("#{Date.today.to_s} 00:00").utc
```

Comment:
IS there any other way to set the hour? I know that that DateTime.parse(string) works, it would be like this:

`DateTime.parse("#{Date.today} 10:00").utc`